m2cgen/exporters.py

```diff
@@ -16,6 +17,8 @@ def export_to_java(model, package_name=None, class_name="Model", indent=4):
         The name of the generated class.
     indent : int, optional
         The size of indents in the generated code.
+    function_name : string, optional
+        Name of the function in the generated code (defaults to 'score')
```

Comment:
< Here and in all other places >
The same thing about writing default value in docstring as above. Also, please add dot at the end of sentence for the consistency with other descriptions.